src/ring_theory/derivation.lean

```diff
@@ -73,6 +73,8 @@ lemma coe_injective : @function.injective (derivation R A M) (A → M) coe_fn :=
 @[ext] theorem ext (H : ∀ a, D1 a = D2 a) : D1 = D2 :=
 coe_injective $ funext H
 
+lemma congr_der (h : D1 = D2) (a : A) : D1 a = D2 a := congr_fun (congr_arg coe_fn h) a
```

Comment:
I called it `congr_fun` but someone in the previous PR asked me to call it this way and I accepted because I know nothing about naming conventions. I cannot find that comment now, but maybe it was @eric-wieser ? In case not I will look for it more.

Reply:
Not me. `congr_fun` is indeed the usual convention here, but someone more familiar with `derivation` than me might have had a reason to diverge.

Reply:
Let's just change it back then. I can only access through my phone today and I'ts impossible I will find that comment. It's not a big deal to change it back if it is wrong